pkg/cloud/aws/regions/regions.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package regions
+
+import "os"
+
+// Default is the region used when no region is specified.
+var Default string
+
+func init() {
+	if Default = os.Getenv("AWS_DEFAULT_REGION"); Default == "" {
```

Comment:
Instead of creating a new variable here, would it make sense to use `AWS_REGION` here instead of `AWS_DEFAULT_REGION`? We don't yet have support for specifying specific regions and end-users would have to update the aws-provider-controller-manager StatefulSet definition to set this env var, which might be more intuitive if it aligns with the standard AWS environment variables.

Reply:
Hi @detiber,

So `AWS_DEFAULT_REGION` is the standard value. Please see https://github.com/aws/aws-sdk-go/issues/2103 for more information.

Reply:
Hi @detiber,

I am happy to support the fallback and support `AWS_REGION` if `AWS_DEFAULT_REGION` is not set.

Reply:
TIL, I'm not sure how I never came across that before.